proto/pdpb.proto

```diff
@@ -295,3 +295,35 @@ message Error {
     optional BootstrappedError     bootstrapped = 2;
     optional StoreIsTombstoneError is_tombstone = 3;
 }
+
+service PlacementService {
+    rpc Tso(Request) returns (Response) {}
```

Comment:
For better understand, I prefer TsoRequest, but it may break the backward compatibility.